src/novel_applications/payment/private_payment.md

```diff
@@ -0,0 +1,167 @@
+
+These issues have prevented blockchains from acquiring the role of leading payment infrastructure and forced users to continue relying on classical centralized infrastructures (e.g., Visa, Mastercard, PayPal, Venmo, Revolut, banks, etc.).
+
+This raises the question: *What about current private decentralized systems?*
+
+While decentralized privacy-preserving protocols like [Zcash](https://z.cash) and [Monero](https://getmonero.org) exist, they face significant limitations. These systems lack stable assets, exclusively using their native tokens (ZEC and XMR), which prevents the use of widely adopted stablecoins like USDC or USDT. This absence of stable, pegged currencies hinders real-world transactions and broad adoption. Additionally, their transaction fees (Zcash: $0.023, Monero: $0.061), while seemingly low, are still too high to compete effectively with traditional payment systems or newer blockchain solutions.
+
+Miden solves these issues.
+
+## Miden: an improved payment infrastructure
+
+Given the limitations of existing blockchains and payments infrastructure, both public and privacy-focused, there is a clear need for a solution that combines privacy, efficiency, and functionality. This is where Miden enters the picture. By building on the foundational work around privacy of projects like Zcash and Monero and around computation of a project like Ethereum, while also incorporating lessons learned from scalable solutions like rollups, Miden presents a promising approach towards building a more ideal payments infrastructure:
+
+| Aspect | Current Systems | Miden |
+|--------|-----------------|-------|
+| Network Structure | Centralized | Decentralized (On the roadmap) |
+| Asset Support | Lack of stable assets | Arbitrary assets |
+| Privacy | Public | Private |
+| Cost | Expensive | Cheap |
+| Speed | Slow | Fast |
+| User Experience | Poor UX | Abstracted UX (Applications) |
+
+The Miden protocol gives us the best of all worlds by addressing each of the key issues identified in current systems.
+
+## Payments using the Miden protocol
+
+Miden enables users to perform [peer-to-peer private transfers](https://polygon.technology/blog/polygon-miden-transaction-model-2) using local execution and local proving. Here is an example of it step by step:
+
+We agree on the following initial state:
+
+- There are 2 users, Alice and Bob
+- Alice owns 1 Ether, Bob owns no assets
+- Alice wants to send her 1 Ether to Bob
+- Alice and Bob want to remain private
+
+To do so, the following scheme can be applied:
+
+1. Alice transitions her state from a state `S` where she has `1 Ether` to a state `S'` where she has `0 Ether`, transferring the asset into a `Note` using the [P2ID script](https://github.com/0xPolygonMiden/miden-base/blob/main/miden-lib/asm/note_scripts/P2ID.masm).
+
+2. Alice transfers this `Note` containing the asset to Bob using arbitrary solutions (On-chain encrypted notes (on the roadmap), Telegram, Signal, etc.).
```

Comment:
Maybe name "On-chain encrypted notes (on the roadmap)," as the third option and "Signal" and then "Email"? This is a minor point but given the recent discussions around Telegram ...